data/test/scenarios/test_assert.cfg

```diff
@@ -27,3 +27,167 @@
         {RETURN ([not][true][/true][/not])}
     [/event]
 )}
+
+# Same as the above three but using the Lua unit_test module
+{GENERIC_UNIT_TEST "test_lua_assert" (
+    [event]
+        name = start
+        [lua]
+            code=<<
+                unit_test.assert(true, 'assert true')
+                unit_test.pass_if(true, 'return true')
+            >>
+        [/lua]
+    [/event]
+)}
+{GENERIC_UNIT_TEST "test_lua_assert_fail" (
+    [event]
+        name = start
+        [lua]
+            code=<<
+                unit_test.assert(false, 'assert false')
+                unit_test.pass_if(true, 'return true')
+            >>
+        [/lua]
+    [/event]
+)}
+{GENERIC_UNIT_TEST "test_lua_assert_fail_two" (
+    [event]
+        name = start
+        [lua]
+            code=<<
+                unit_test.assert(true, 'assert true')
+                unit_test.pass_if(false, 'return false')
+            >>
+        [/lua]
+    [/event]
+)}
+
+# Test the comparison assertions from the Lua unit_test module
+{GENERIC_UNIT_TEST "test_lua_assert_compare" (
+    [event]
+        name=start
+        [lua]
+            code=<<
+                unit_test.assert_equal('foo', 'foo', 'string equality')
+                unit_test.assert_not_equal('foo', 'bar', 'string inequality')
+                unit_test.assert_greater('foo', 'bar', 'string ordering')
+                unit_test.assert_equal(1, 1, 'numeric equality')
+                unit_test.assert_not_equal(1, 2, 'numeric inequality')
+                unit_test.assert_greater(2, 1, 'numeric greater-than')
+                unit_test.assert_less(1, 2, 'numeric less-than')
+                unit_test.assert_in_range(1, 0, 2, 'numeric in-range')
+                unit_test.assert_approx_equal(1, 1.0001, 0.0001, 'numeric approx-equal')
```

Comment:
Here the difference between the two values is the same as the delta, so the result of this should be uncertain.

Instead you should test a value that's certain - such as `(1, 1.0001, 0.001)`, where the difference is an order of magnitude less than the delta.